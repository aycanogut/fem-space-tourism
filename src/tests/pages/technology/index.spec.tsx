import renderer from 'react-test-renderer'
import Technology from '../../../pages/technology'

it('renders correctly', () => {
  const tree = renderer.create(<Technology />).toJSON()
  expect(tree).toMatchSnapshot()
})
