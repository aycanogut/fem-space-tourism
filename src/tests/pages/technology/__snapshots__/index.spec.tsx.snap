// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
[
  <header
    className="relative z-20 h-0 w-full"
    style={
      {
        "transform": "translateY(-100px) translateZ(0)",
      }
    }
  >
    <div
      className="flex h-full items-center justify-between px-6 sm:px-10 lg:px-14"
    >
      <a
        className="focus:outline-none"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        <svg
          className="mt-nav_sm_top h-10 w-10 sm:mt-nav_md_top sm:h-12 sm:w-12 lg:mt-nav_lg_top"
          style={
            {
              "display": "inline-block",
              "fill": "currentColor",
              "stroke": "currentColor",
            }
          }
          viewBox="0 0 1024 1024"
        >
          <path
            d="M1024 512c0 282.77-229.23 512-512 512s-512-229.23-512-512c0-282.77 229.23-512 512-512s512 229.23 512 512z"
            fill="rgb(255, 255, 255)"
          />
          <path
            d="M512 0c0 341.333-170.667 512-512 512 335.317 2.432 505.984 173.099 512 512 0-341.333 170.667-512 512-512-341.333 0-512-170.667-512-512z"
            fill="rgb(11, 13, 23)"
          />
        </svg>
      </a>
      <svg
        className="visible mt-nav_sm_top sm:invisible sm:mt-nav_md_top"
        height="21px"
        onClick={[Function]}
        style={
          {
            "display": "inline-block",
            "fill": "currentColor",
            "stroke": "currentColor",
          }
        }
        viewBox="0 0 1170 1024"
        width="24px"
      >
        <path
          d="M0 0h1170.286v146.286h-1170.286zM0 438.857h1170.286v146.286h-1170.286zM0 877.714h1170.286v146.286h-1170.286z"
          fill="rgb(208, 214, 249)"
        />
      </svg>
    </div>
    <nav
      className="absolute top-0 right-0 h-screen w-2/3 bg-white bg-opacity-5 backdrop-blur-nav before:absolute before:-left-[30%] before:top-1/2 before:hidden before:h-[1px] before:w-1/3 before:bg-white before:opacity-30 sm:ml-auto sm:h-24 sm:w-nav_sm lg:mt-10 lg:w-nav_lg before:lg:block"
      role="navigation"
      style={
        {
          "opacity": 0,
        }
      }
    >
      <div
        className="flex h-full flex-col items-end pl-8 pr-6 pt-9 sm:pt-0 sm:pr-0 sm:pl-0"
      >
        <svg
          className="cursor-pointer sm:!hidden"
          height="21px"
          onClick={[Function]}
          style={
            {
              "display": "inline-block",
              "fill": "currentColor",
              "stroke": "currentColor",
            }
          }
          viewBox="0 0 975 1024"
          width="24px"
        >
          <path
            d="M125.562 46.519l827.49 827.49-103.375 103.473-827.49-827.49z"
            fill="rgb(208, 214, 249)"
          />
          <path
            d="M22.138 874.057l827.49-827.49 103.473 103.375-827.49 827.49z"
            fill="rgb(208, 214, 249)"
          />
        </svg>
        <ul
          className="mr-auto flex flex-col gap-9 pt-14 sm:ml-auto sm:h-full sm:flex-row sm:items-center sm:pt-0 lg:ml-nav_lg_left lg:mr-auto lg:gap-12"
        >
          <a
            className=" flex h-full items-center text-white transition-all focus:border-white focus:outline-none md:hover:border-b-[3px] md:hover:border-white/[.5] md:active:border-white"
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <li
              className="mt-[2px] flex gap-3 font-barlow text-16 uppercase tracking-normal sm:mt-0 sm:text-14 lg:gap-nav_lg_gap lg:text-16"
            >
              <span
                className="font-bold sm:hidden lg:block"
              >
                00
              </span>
              <span
                className="font-normal"
              >
                home
              </span>
            </li>
          </a>
          <a
            className=" flex h-full items-center text-white transition-all focus:border-white focus:outline-none md:hover:border-b-[3px] md:hover:border-white/[.5] md:active:border-white"
            href="/destination"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <li
              className="mt-[2px] flex gap-3 font-barlow text-16 uppercase tracking-normal sm:mt-0 sm:text-14 lg:gap-nav_lg_gap lg:text-16"
            >
              <span
                className="font-bold sm:hidden lg:block"
              >
                01
              </span>
              <span
                className="font-normal"
              >
                destination
              </span>
            </li>
          </a>
          <a
            className=" flex h-full items-center text-white transition-all focus:border-white focus:outline-none md:hover:border-b-[3px] md:hover:border-white/[.5] md:active:border-white"
            href="/crew"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <li
              className="mt-[2px] flex gap-3 font-barlow text-16 uppercase tracking-normal sm:mt-0 sm:text-14 lg:gap-nav_lg_gap lg:text-16"
            >
              <span
                className="font-bold sm:hidden lg:block"
              >
                02
              </span>
              <span
                className="font-normal"
              >
                crew
              </span>
            </li>
          </a>
          <a
            className=" flex h-full items-center text-white transition-all focus:border-white focus:outline-none md:hover:border-b-[3px] md:hover:border-white/[.5] md:active:border-white"
            href="/technology"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <li
              className="mt-[2px] flex gap-3 font-barlow text-16 uppercase tracking-normal sm:mt-0 sm:text-14 lg:gap-nav_lg_gap lg:text-16"
            >
              <span
                className="font-bold sm:hidden lg:block"
              >
                03
              </span>
              <span
                className="font-normal"
              >
                technology
              </span>
            </li>
          </a>
        </ul>
      </div>
    </nav>
  </header>,
  <main
    className="flex w-screen bg-cover bg-center undefined"
    style={
      {
        "backgroundImage": "url(/assets/technology/background-technology-desktop.webp)",
      }
    }
  />,
]
`;
